kolibri/plugins/learn/assets/src/views/TopicsPage/HeaderTab.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+
+  <!-- Stolen from Coach -->
+  <router-link
+    ref="btn"
+    :to="to"
+    class="header-tab"
+    :activeClass="activeClasses"
+    :style="{ color: $themeTokens.annotation }"
+    :replace="true"
+    :class="defaultStyles"
+  >
+    <div class="inner" :style="{ borderColor: this.$themeTokens.primary }">
+      {{ text }}
+    </div>
+  </router-link>
+
+</template>
+
+
+<script>
+
+  export default {
+    name: 'HeaderTab',
+    props: {
+      text: {
+        type: String,
+        required: true,
+      },
+      to: {
+        type: Object,
+        required: true,
+      },
+    },
+    computed: {
+      activeClasses() {
+        // return both fixed and dynamic classes
+        return `router-link-active ${this.$computedClass({ color: this.$themeTokens.primary })}`;
+      },
+      defaultStyles() {
+        return this.$computedClass({
+          ':focus': this.$coreOutline,
+          ':hover': {
+            backgroundColor: this.$themePalette.grey.v_300,
+          },
+        });
+      },
+    },
+  };
+
+</script>
+
+
+<style lang="scss" scoped>
+
+  @import '~kolibri-design-system/lib/styles/definitions';
+
+  // a lot copied from KButton
```

Comment:
I was wondering where all the extra styles were coming from here.

At the risk of dog piling - is it worth checking with @jtamiace who I know had a couple of regrets about how these header links were done, and I think slightly redesigned them for 0.16. Maybe we could just do that directly here instead?

Reply:
I think I'll hold off for now and wait for the redesigns to come through as the header tab designs may be further impacted by design decisions and they're not finalized yet.